src/device/usbd.c

```diff
@@ -658,13 +657,17 @@ static bool process_control_request(uint8_t rhport, tusb_control_request_t const
         }
       }
 
-      // Some classes such as USBTMC needs to clear/re-init its buffer when receiving CLEAR_FEATURE request
-      // We will forward all request targeted endpoint to class drivers after
-      // - For class-type requests: driver is fully responsible to reply to host
-      // - For std-type requests  : driver init/re-init internal variable/buffer only, and
-      //                            must not call tud_control_status(), driver's return value will have no effect.
-      //                            EP state has already affected (stalled/cleared)
-      if ( invoke_class_control(rhport, drvid, p_request) ) ret = true;
+      if (ep_num != 0 || (ep_num == 0 && drvid != 0xFF)) {

```

Comment:
I think the if condition is a bit overkill, a simple
`if ( drvid < USBD_CLASS_DRIVER_COUNT )`
is sufficient enough. Else we just simply  do nothing :)

Reply:
Ok, changed as you wish. This is what I had originally but thought it might be a bit too broad.